qutebrowser/utils/utils.py

```diff
@@ -776,14 +777,38 @@ def mimetype_extension(mimetype: str) -> Optional[str]:
 
     This mostly delegates to Python's mimetypes.guess_extension(), but backports some
     changes (via a simple override dict) which are missing from earlier Python versions.
-    Most likely, this can be dropped once the minimum Python version is raised to 3.10.
     """
-    overrides = {
-        # Added in 3.10
-        "application/x-hdf5": ".h5",
-        # Added around 3.8
-        "application/manifest+json": ".webmanifest",
-    }
+    overrides = {}
```

Comment:
Decided to do those conditionally based on the Python version where CPython added them, for two reasons:

- I saw that sometimes CPython *fixes* existing mimetypes, in which case we would undo the fix with an outdated override
- Tools like `ruff` can detect and remove such `if sys.version_info` blocks when telling it to upgrade things, so we won't forget to adjust this once we drop Python 3.9.